tests/system/action/motion/test_reset_recommendation.py

```diff
@@ -0,0 +1,36 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MotionResetRecommendationActionTest(BaseActionTestCase):
+    def test_reset_recommendation_correct(self) -> None:
+        self.create_model("meeting/222", {"name": "name_SNLGsvIV"})
+        self.create_model(
+            "motion_state/77",
+            {"name": "test1", "motion_id": 22, "motion_recommendation_ids": [22]},
+        )
+        self.create_model(
+            "motion/22", {"meeting_id": 222, "title": "test1", "recommendation_id": 77}
+        )
+        response = self.client.post(
+            "/", json=[{"action": "motion.reset_recommendation", "data": [{"id": 22}]}]
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/22")
+        assert model.get("recommendation_id") is None
+
+    def test_reset_recommendation_correct_2(self) -> None:
```

Comment:
That is exactly the same test as the first one, or am I missing something here? I'm guessing c&p error. Anyway, a negative test is missing.

Reply:
No, it is not: In the first test the recommendation_id was set to a state, in the second, it was set to None.

Reply:
Ah, okay, missed that.